src/components/templates/Jazzbar/JazzTab/JazzTab.tsx

```diff
@@ -251,7 +251,7 @@ const Jazz: React.FC<JazzProps> = ({ setUserList, venue }) => {
         </div>
         <UserList
           isAudioEffectDisabled={isAudioEffectDisabled}
-          users={venueUsers}
+          users={recentWorldUsers}
           activity={venue?.activity ?? "here"}
           disableSeeAll={false}
         />
```

Comment:
This seems like it's going to change the logic/intended effect, as `usePartygoers` filtered down to users that had been in this venue recently, whereas `recentWorldUsers` doesn't.

This would be resolved by implementing `useRecentVenueUsers(venueId)` as mentioned elsewhere in these review comments.